bundles/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComRainMessage.java

```diff
@@ -38,7 +38,9 @@
         RAIN4(4),
         RAIN5(5),
         RAIN6(6),
-        RAIN7(7);
+        RAIN7(7),
```

Comment:
Could 7 and 8 be added also consistently?

Reply:
Unfortunately, the documentation does not state to which devices these types belong. Neither does any of the patch notes.
Would you prefer to introduce placeholder names for them in .../thing/rain.xml?

Reply:
No let's not introduce placeholders. I'll contact Bert about a new version of the SDK documentation

Reply:
I wasn't aware of the SDK. I was referring to the User Manual.
A quick glance at the implementation of the protocol in  [node-rfxcom](https://github.com/rfxcom/node-rfxcom/blob/master/lib/rfxcom.js) shows that RAIN8 belongs to Davis rain gauges (according to the comments in the code).
I also realized that the calibration of the meter is off. node-rfxcom implements a special case for RAIN9 that corrects this.
How can I get access to the SDK documentation? I would be willing to implement the necessary changes myself.

Reply:
@muhl I get and got the SDK by contacting the person behind RFXCOM, but I cannot just pass it, however feel free to contact him.

The RAIN6, RAIN8, and RAIN9 require specific logic in the code so if you have the time to add that support that would be great.